### PR TITLE
Update the eye tracking documentation to provide information on the new Gaze Input capability helper of the MRTK build tools.

### PR DESCRIPTION
**Pull request by wiwei**
It's now possible to have the MRTK build tools add the capability for you automatically, so calling this out on the documentation is helpful.

### CODE REVIEW

File: Documentation/EyeTracking/EyeTracking_BasicSetup.md [hunk 1]
```diff
@@ -12,11 +12,24 @@ We will go into detail on how to address each of them further below.
...
+### A note on the GazeInput capability
+
+If you are using the MRTK-provided build tooling (i.e. by going to the Mixed Reality Toolkit -> Utilities -> Build Window)
+to build the AppX, the MRTK build tools can automatically enable the GazeInput capability for you. In order to this,
+you need to make sure that the 'Gaze Input Capability' is checked on the 'Appx Build Options' tab:
+
+![MRTK Build Tools](../Images/EyeTracking/mrtk_et_buildsetup.png)
+
+This tooling will find the AppX manifest after the Unity build is completed and manually add the GazeInput capability.
+**Note that this tooling is NOT active if you are using Unity's built-in Build Window** (i.e. File -> Build Settings).
```
Comment (keveleigh) [inline Documentation/EyeTracking/EyeTracking_BasicSetup.md]: Would it be possible to hook into Unity's build tooling with https://docs.unity3d.com/ScriptReference/Build.IPostprocessBuildWithReport.OnPostprocessBuild.html to get it working either way?

File: Documentation/EyeTracking/EyeTracking_BasicSetup.md [hunk 2]
```diff
@@ -12,11 +12,24 @@ We will go into detail on how to address each of them further below.
...
-3. The _'GazeInput'_ capability must be enabled in the application manifest. **Currently this is only available in Visual Studio.**
+3. The _'GazeInput'_ capability must be enabled in the application manifest. 
+   **Currently this is only available in Visual Studio and through the MRTK build tool**
 4. The HoloLens **must** be eye calibrated for the current user. Check out our [sample for detecting whether a user is eye calibrated or not](EyeTracking_IsUserCalibrated.md). 
 
 **IMPORTANT:** If any of the above requirements are not met, the application will automatically fall back to head-based gaze tracking.
 
+### A note on the GazeInput capability
+
+If you are using the MRTK-provided build tooling (i.e. by going to the Mixed Reality Toolkit -> Utilities -> Build Window)
```
Comment (thalbern) [inline Documentation/EyeTracking/EyeTracking_BasicSetup.md]: we should avoid using "you" in the documentation:
While using the MRTK-provided build tooling for building the AppX, the MRTK build tools can automatically......

File: Documentation/EyeTracking/EyeTracking_BasicSetup.md (hunk 1 above)
Comment (wiwei) [reply Documentation/EyeTracking/EyeTracking_BasicSetup.md]: Yes, this was a possibility, but it was a level of automagic that I wasn't really comfortable with.

**Pull request merged**